src/metrics.cc

```diff
@@ -34,49 +29,35 @@ Metrics* g_metrics = NULL;
 
 namespace {
 
-#ifndef _WIN32
 /// Compute a platform-specific high-res timer value that fits into an int64.
 int64_t HighResTimer() {
```

Comment:
With the assumption that the C++11 PR gets merged, is there a reason not to have HighResTimer return the result of chrono::steady_clock::now() directly? E.g. why convert to int at all?

Reply:
For that matter, why bother with this function anyway? E.g. just replace everywhere it's called with std::chrono::steady_clock::now()